extension/snippets/tests.json

```diff
@@ -113,5 +113,29 @@
       "\tend;"
     ],
     "description": "Snippet: Test ConfirmHandler"
+  },
+  "Snippet: GIVEN clause": {
+    "prefix": "tGIVEN (NAB)",
+    "body": [
+      "// [GIVEN] ${1:Given}",
+      "${0}"
+    ],
+    "description": "Snippet: GIVEN test clause"
+  },
+  "Snippet: WHEN clause": {
+    "prefix": "tWHEN (NAB)",
+    "body": [
+      "// [WHEN] ${1:When}",
+      "${0}"
+    ],
+    "description": "Snippet: WHEN test clause"
+  },
+  "Snippet: THEN clause": {
+    "prefix": "tTHEN (NAB)",
+    "body": [
+      "// [THEN] ${1:Then}",
+      "${0}"
+    ],
+    "description": "Snippet: THEN test clause"
```

Comment:
oooh! It has been a while since we added snippets :star_struck:  How about adding an all in one?

```json
  "Snippet: GIVEN/WHEN/THEN": {
    "prefix": "tGIVEN/WHEN/THEN (NAB)",
    "body": [
      "// [GIVEN] ${1:Given}",
      "// [WHEN] ${2:When}",
      "// [THEN] ${3:Then}",
      "${0}"
    ],
    "description": "Snippet: GIVEN/WHEN/THEN test clauses"
  }

Reply:
We already got them together in the `ttestfunction` snippet, but I don't really like it the way it is there... Since it's (most) often many GIVEN and many THEN, it makes no sense to first fill in one GIVEN, WHEN, THEN and after that go back and add GIVENs and THENs.

At least for me, it's more natural to use the new snippets to add all GIVENs, one WHEN and then (😜) all THENs.